adb_graphics/datahandler/gribfile.py

```diff
@@ -127,6 +127,16 @@ def free_fcst_names(self, ds, fcst_type):
                         suffix == f'{suf}1h':
                         contains_suffix.append(suf)
 
+                    # RRFS_A has fields that have the suffix 'acc0h' but we don't
```

Comment:
If these fields are based on my (CSH) suggestions, there may be some missing or some that don't need to be in the list.  We'll need to monitor the list as more fields are added to the grib files and requested for graphics.

Reply:
Yes, we should monitor the list.  Hopefully, it won't need to be changed for a long time.